feature-libs/asm/root/config/asm-config.ts

```diff
@@ -13,6 +15,19 @@ export abstract class AsmConfig extends OccConfig {
     customerSearch?: {
       maxResults?: number;
     };
+    customerList?: {
```

Comment:
Why some of the asm config is moved from core to root?

Reply:
I can explain this.
For this PR (https://github.com/SAP/spartacus/pull/16095) I added an interceptor `UserIdHttpHeaderInterceptor` that lived in `@spartacus/asm/root`.
As it needed to be turned on/off - depending on the customer's backend - a configuration was added to `AsmConfig`.
This caused a circular dependency - `@spartacus/asm/root` depended on `@spartacus/asm/core` which depends on `root`.
Patrick and I concluded the configuration should have been in `root` originally. There is precedence for this - for example, `CartConfig` is in root.
Let me know if you have any further questions.